identity-and-access-management/audit-trail/concepts.mdx

```diff
@@ -0,0 +1,45 @@
+---
+meta:
+  title: Audit Trail - Concepts
+  description: Learn fundamental concepts of Scaleway's Audit Trail.
+content:
+  h1: Concepts
+  paragraph: Learn fundamental concepts of Scaleway's Audit Trail.
+tags: audit-trail log events access
+dates:
+  validation: 2024-11-26
+---
+
+## Audit Trail
+
+Audit Trail is a tool that holds a record of events and changes performed within a Scaleway Organization. Audit Trail allows you to keep track of all actions, whether successful, attempted, or failed, perfomed on your Scaleway resources.
+
+With Audit Trail you can view actions taken by any [principal](/identity-and-access-management/iam/concepts/#principal) in any Project and on any of your Scaleway resources. You can also see the API methods used, the outcome of the action performed (status), and the IP address from which a principal performed an action.
+
+Audit Trail allows you to view the metadata of the API method used, allowing you to investigate incidents and troubleshoot issues.
+
+## Event
+
+An event is a log of user activity on your Scaleway resources.
+
+## Locacion
+
+In the context of Audit Trail, location refers to the place where the Scaleway resources being monitored are located. They can be located in different Scaleway Projects, or Organizations.
+
+## Method
+
+A method in the context of Audit Trail is the API call sent to the Audit Trail API whenever a principal takes an action (whether successful, attempted, or failed) on any Scaleway resource.
+
+## Region
+
+A region is the geographical area in which your resources are located. They are duplicated on all [Availability Zones](/compute/instances/concepts/#availability-zone) of the selected region (Paris, Amsterdam, or Warsaw).
```

Comment:
A region is the geographical area in which your resources are located only for regional resources... but Secret Manager is one, and k8s too.

However, we should remove the sentence "they are duplicated on all..." because it is not always true fir regionalized products, and certainly is not for Secret Manager